.github/workflows/reusable-prober.yml

```diff
@@ -87,7 +87,12 @@ jobs:
       - name: Run prober
         env:
           DEBUG: 1
-        run: prober --one-time --rekor-url ${{ inputs.rekor_url }} --fulcio-url ${{ inputs.fulcio_url }}
+        uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd # v2.8.3
```

Comment:
If we want to try running this only in staging to start, we could add a new job step that uses the `retry` action and leave this one untouched. We could then use an input to signify whether we want to use the existing `Run prober` job step or the new experimental one.